### PR TITLE
Add Login functionality

### PR DESCRIPTION
**Pull request by jesspearson**
-Added a front-end login page.
-Updated PagesController file with a login function.
-Created a bloggerLogin class/model.
-Updated routes file to include 'login' action to the 'pages' controller.
-Added 'Log in' link to existing nav in layout.php
-Finally have added a placeholder 'loggedInPage' to navigate to after successful log in. This will need to be replaced once the write blog page is added.

### CODE REVIEW

File: models/bloggerLogin.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * This class contains the grantAccess() function which we use to 
+ * determine if the user has entered valid login credentials
+ *
+ * @author jess
+ */
+class bloggerLogin {
```
Comment (ClareMDavidson) [inline models/bloggerLogin.php:9]: Should we have all our classes starting with capital letters? Apparently that's the stylistic norm?

File: models/bloggerLogin.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+// COMMENTED OUT CONSTRUCTOR AS DON"T THINK WE ACTUALLY NEED ONE FOR THIS?    
+//    public $username;
+//    public $password;
+//    
+//    public function __construct($username, $password) {
+//        $this->username = $username;
+//        $this->password = $password; 
+//    }
+    
+    const BLOGUSERNAME = 'BLOGGER';
```
Comment (ClareMDavidson) [inline models/bloggerLogin.php:20]: Should all the constants go in one place?
Would you like to start constants.php? Then you'd need to include that file in this one to use the constants.

File: models/bloggerLogin.php (hunk 1 above)
Comment (jesspearson) [reply models/bloggerLogin.php:9]: Updated this

File: models/bloggerLogin.php (hunk 2 above)
Comment (jesspearson) [reply models/bloggerLogin.php:20]: Yeh possibly, maybe if more and more constants are getting added then we can look at breaking them out into their own file.

**Pull request merged**